src/Core/Mews.Fiscalizations.Core/Xml/XmlSerializer.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Serialization;
+using FuncSharp;
+using Mews.Fiscalizations.Core.Model;
+
+namespace Mews.Fiscalizations.Core.Xml
+{
+    public sealed class XmlSerializer
+    {
+        public static XmlElement Serialize<T>(T value, XmlSerializationData serializationData = null)
+            where T : class
+        {
+            var namespaceSerializer = new XmlSerializerNamespaces();
+            var namespaces = serializationData.ToOption().Match(
+                d => d.Namespaces.Flatten(),
+                _ => Enumerable.Empty<XmlNamespace>()
+            );
```

Comment:
Should be possible like this, right?

```suggestion
            var namespaces = serializationData.ToOption().FlatMap(d => d.Namespaces).Flatten());
```